pkgs/development/python-modules/slackclient/default.nix

```diff
@@ -1,29 +1,55 @@
-{ stdenv, buildPythonPackage, fetchFromGitHub, websocket_client, requests, six, pytest, codecov, coverage, mock, pytestcov, pytest-mock, responses, flake8 }:
+{ stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, aiohttp
+, black
+, codecov
+, flake8
+, mock
+, pytest-mock
+, pytestCheckHook
+, pytestcov
+, pytestrunner
+, requests
+, responses
+, six
+, websocket_client
+}:
 
 buildPythonPackage rec {
   pname = "python-slackclient";
-  version = "1.2.1";
+  version = "2.5.0";
 
   src = fetchFromGitHub {
     owner  = "slackapi";
     repo   = pname;
     rev    = version;
-    sha256 = "073fwf6fm2sqdp5ms3vm1v3ljh0pldi69k048404rp6iy3cfwkp0";
+    sha256 = "1ngj1mivbln19546195k400w9yaw69g0w6is7c75rqwyxr8wgzsk";
   };
 
```

Comment:
Add `disabled=isPy27` to remove Python 2.7 support

Reply:
Used `isPy3k` instead.